pgbouncer-mixin/main.libsonnet

```
@@ -0,0 +1,35 @@
+local alerts = import './alerts.libsonnet';
+local config = import './config.libsonnet';
+local dashboards = import './dashboards.libsonnet';
+local g = import './g.libsonnet';
+local links = import './links.libsonnet';
+local panels = import './panels.libsonnet';
+local targets = import './targets.libsonnet';
+local variables = import './variables.libsonnet';
+
+{
+
+  withConfigMixin(config): {
+    config+: config,
+  },
+
+  new(): {
+
+    local this = self,
+    config: config,
+
+    grafana: {
+      variables: variables.new(this, varMetric='up{%(filteringSelector)s}' % this.config),
```

```suggestion
      variables: variables.new(this, varMetric='up')
```
I think filteringSelector is added in variables.